Userland/Libraries/LibGUI/Label.h

```diff
@@ -36,6 +37,9 @@ class Label : public Frame {
     bool is_word_wrap() const { return m_word_wrap; }
     void set_word_wrap(bool);
 
+    GUI::WordWrapMode word_wrap_mode() const { return m_word_wrap_mode; }
```

Comment:
This can be "full const" if I see that correctly.
```suggestion
    GUI::WordWrapMode const word_wrap_mode() const { return m_word_wrap_mode; }
```

Reply:
I get some compile errors with this change, is it really necessary? `text_alignment()` doesn't have it and both getters have the same logic..

Reply:
Nevermind I guess